lms/djangoapps/certificates/api.py

```diff
@@ -466,7 +466,10 @@ def get_language_specific_template(language, templates):
     return None
 
 
-def get_all_languages_or_default_template(templates):  # lint-amnesty, pylint: disable=missing-function-docstring
+def get_all_languages_or_default_template(templates):
+    """
+    Returns the first template whose language is an empty string
```

Comment:
nit: "Returns the first template that isn't language specific" or something more about the goal of the function rather than the implementation.